packages/ui-components/src/components/StyleProvider/Fonts.js

```diff
@@ -11,39 +11,27 @@
 import React, { useInsertionEffect } from "react"
 import PropTypes from "prop-types"
 
-const CSS_FONTS_URL = "https://assets.juno.global.cloud.sap/assets/fonts/plex/css/ibm-plex.min.css"
+import fonts from "../../fonts/plex/css/ibm-plex.scss"
 
 const STYLE_ID = "juno-style-provider-golbal-fonts"
 const Fonts = ({ inline }) => {
   // add fonts to document.head
   useInsertionEffect(() => {
+    if (inline) return
+
     // add necessary prerequisites to HEAD to use google fonts
     if (!document.querySelector(`[id="${STYLE_ID}"]`)) {
-      // Add font links to head (Plex font from google CDN)
-      const link1 = document.createElement("link")
-      link1.rel = "preconnect"
-      link1.href = "https://fonts.googleapis.com"
-
-      const link2 = document.createElement("link")
-      link2.rel = "preconnect"
-      link2.href = "https://fonts.gstatic.com"
-      link2.crossOrigin = "anonymous"
-
-      const link3 = document.createElement("link")
```

Comment:
I believe we need this link. Afaik in order for webfonts to work properly within the shadow root they need to be included via link in the head (which is what this `link3` does) and additionally inside of the shadow root they need to be included again inline (which is what happens in line 34). 

This might be outdated information, but please double check again.

Reply:
I'll check, thanks

Reply:
Since we are now embedding the fonts as text, in my opinion, they no longer need to be loaded separately in the head. Currently, we either load the fonts inline in the shadow root as text or in the head.